addon/sidebar.css

```diff
@@ -60,39 +60,53 @@ body {
   display: flex;
   flex-direction: column;
   margin-bottom: 16px;
+  width: 71vw;
+  max-width: 284px;
+  height: 52vw;
+  max-height: 206px;
 }
 
 .graphic__overlay {
   position: relative;
-  width: 284px;
-  height: 174px;
-  margin-bottom: 16px;
+  max-width: 284px;
+  width: 100%;
+  height: 61vw;
+  max-height: 174px;
+  margin-bottom: 9.2%;
 }
 
 .graphic__clouds {
-  width: 284px;
-  height: 174px;
+  max-width: 284px;
+  width: 100%;
+  max-height: 174px;
+  height: 100%;
   background: url(images/in-content-clouds.png);
-  background-size: 284px 174px;
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-position: center;
```

Comment:
background: url(images/in-content-clouds.png) center no-repeat;

And below. (contain is supposed to be able to go in the one-liner too but it's misbehaving"